src/main/java/org/terasology/biomesAPI/Biome.java

```diff
@@ -62,17 +50,16 @@ default float getTemperature() {
     }
 
     /**
-     * @return The block that should be generated as the top layer of the biome at the given position.
-     * Defaults to grass.
+     * @return The block that should be generated as the top layer of the biome at the given position. Defaults to grass.
      */
     default Block getSurfaceBlock(Vector3ic pos, int seaLevel) {
         BlockManager blockManager = CoreRegistry.get(BlockManager.class);
         return blockManager.getBlock("CoreAssets:Grass");
     }
 
     /**
-     * @return The block that should be generated at the given position in the biome, which isn't the surface.
-     * Defaults to dirt for the first 32 blocks and then stone below.
+     * @return The block that should be generated at the given position in the biome, which isn't the surface. Defaults to dirt for the
+     *         first 32 blocks and then stone below.
```

Comment:
on purpose or accidental?
```suggestion
     * first 32 blocks and then stone below.
```

Reply:
if you want to leave it feel free. just used intellij default format, shift-ctrl-alt-k and it made it so ... @jdrueckert

Reply:
if it's on purpose and auto-indented by intellij, that's fine 👍